osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -169,7 +169,6 @@ private void load(BeatmapDatabase beatmaps, AudioManager audio, DialogOverlay di
 
             if (osu != null)
                 playMode.BindTo(osu.PlayMode);
-            playMode.ValueChanged += val => Beatmap.PreferredPlayMode = val;

```

Comment:
also please don't merge this yet; it doesn't completely address issues surrounding this binding